packages/@react-spectrum/tabs/stories/Tabs.stories.tsx

```diff
@@ -73,6 +73,18 @@ storiesOf('Tabs', module)
     'icons, density: compact, orientation: vertical',
     () => renderWithIcons({orientation: 'vertical', density: 'compact'})
   )
+  .add(
+    'isEmphasized: true',
+    () => render({isEmphasized: true})
+  )
+  .add(
+    'isEmphasized: true, icons, isQuiet: true',
+    () => renderWithIcons({isEmphasized: true, isQuiet: true})
+  )
+  .add(
```

Comment:
Might be good to throw in some with `desnsity: compact`

Reply:
Added them to chromatic stories.